docs/spec/functional_spec.rst

```diff
@@ -95,52 +96,63 @@ or if the sender is not the burrow owner.
-not the burrow owner. If deactivation is successful, make a tez payment to
-the given address.
+Activate a currently inactive burrow. Fails if the burrow does not exist, if the
+burrow is already active, if the amount of tez given is less than the creation
+deposit, or if the sender is not the burrow owner.
 
-``deactivate_burrow: nat``
+``activate_burrow: nat``
 
 +---------------+-----------------------+-------------------------------------------------------------------------+
 | Parameter     |      Field Type       | Description                                                             |
 +===============+=======================+=========================================================================+
-| id            | nat                   | The caller's ID for the burrow to deactivate                            |
+| id            | nat                   | The caller's ID for the burrow to activate                              |
 +---------------+-----------------------+-------------------------------------------------------------------------+
 
 
-Activate an inactive burrow
----------------------------
+Deactivate a burrow
+-------------------
 
-Activate a currently inactive burrow. Fail if the burrow does not exist,
-if the burrow is already active, if the amount of tez given is less than
-the creation deposit, or if the sender is not the burrow owner.
+Deactivate a currently active burrow. Fails if the burrow does not exist, if it
+is already inactive, if it is overburrowed, if it has kit outstanding, if it
+has collateral sent off to auctions, or if the sender is not the burrow owner.
+If deactivation is successful, make a tez payment to the given address.
 
-``activate_burrow: nat``
+``deactivate_burrow: (pair nat address)``
 
 +---------------+-----------------------+-------------------------------------------------------------------------+
 | Parameter     |      Field Type       | Description                                                             |
 +===============+=======================+=========================================================================+
-| id            | nat                   | The caller's ID for the burrow to activate                              |
+| id            | nat                   | The caller's ID for the burrow to deactivate                            |
++---------------+-----------------------+-------------------------------------------------------------------------+
+| receiver      | address               | The address to send the burrow's collateral and creation deposit to     |
 +---------------+-----------------------+-------------------------------------------------------------------------+
 
 
 Perform burrow maintenance
 --------------------------
 
+Perform maintenance tasks on a burrow (i.e., update it's outstanding kit
+according to the system changes that have taken place since the last time the
+burrow was operated on). Fails if the burrow does not exist.
+
```

Comment:
Maybe it's worth noting that most burrow operations call this implicitly and this is only here as a failsafe mechanism.